examples/gen_test_data/gen_test_data/config.ini.example

```diff
@@ -0,0 +1,34 @@
+; config.ini
+; This is a sample configuration file
+
+[COMMON]
+; The COMMON section provides common values for all other sections.
+; Configure both 'document_folder' and 'document_chunk_size' if you require document splitting.
+; However, if you wish to bypass the document split process, simply provide the 'document_node_file', which is a JSONL file.
+; When all these parameters are configured, the system will primarily use the 'document_node_file'
+documents_folder = "<your-document-folder-abspath>"
+document_chunk_size = 1024
+document_nodes_file = "<your-node-file-abspath>"
+
+; Test data gen flow configs
+flow_folder = "<your-test-data-gen-flow-folder-abspath>" ; There is must one flow.dag.yaml file under this folder as entry
+connection_name = "<your-connection-name>"
```

Comment:
What if user fill in connection name in flow but not here?

Asking cause it looks weird that connection and model_or_deployment_name are provided in different places. (connection: config.ini, model_or_deployment_name: flow.dag.yaml)